### DISCUSSION TITLE
Header Filter

### DISCUSSION THREAD

**Discussion by gabrielaHashdex** [pratik227/quasar-qgrid#131] [General]
Header filter is not working on my application, all others filters are working tho. There is something I may be missing?

    <q-grid
      class="shadow-1"
      flat
      bordered
      title="Employees"
      :data="processedRows"
      :columns="columns"
      :pagination="paginationOptions"
      :loading="loading"
      :header_filter="true"
      :global_search="true"
      :fullscreen="true"

    >

**Comment by pratik227** [comment]
@gabrielaHashdex,

You are missing show_filter:true

```
{
    name: 'name',
    required: true,
    label: 'Dessert (100g serving)',
    align: 'left',
    field: 'name',
    sortable: true,
    show_filter:true,
 }
```

**Reply by Anttonino** [reply]
Good, now worked! But this information not exist in [qgrid ](https://quasar-qgrid.netlify.app/examples) documentation, It would be nice if this was specified in the example code. 

The header filter icon doesn't appears. 

![image](https://github.com/pratik227/quasar-qgrid/assets/99145534/c34be5a7-e58e-4c6d-b05f-f40b9779350e)

The following codes are: 
```vue
    <q-grid
      class="shadow-1"
      flat
      bordered
      title="Employees"
      :data="processedRows"
      :columns="columns"
      :pagination="paginationOptions"
      :loading="loading"
      :header_filter="true"
      :global_search="true"
      :fullscreen="true"

    >

    </q-grid>
```

```js
    const columns = [
       {
         name: 'name',
         align: 'left',
         label: 'Name',
         field: 'name',
         sortable: true,
         show_filter:true
    },
```

**Reply by pratik227** [reply]
```
    extras: [
      // 'ionicons-v4',
      // 'mdi-v5',
      'fontawesome-v5',
      // 'eva-icons',
      // 'themify',
      // 'line-awesome',
      // 'roboto-font-latin-ext', // this or either 'roboto-font', NEVER both!

      'roboto-font', // optional, you are not bound to it
      'material-icons' // optional, you are not bound to it
    ],
```

It uses fontawesome-v5 icon uncomment it in quasar-conf-js
